src/OrbitGl/App.cpp

```diff
@@ -599,11 +599,7 @@ void OrbitApp::OnErrorEnablingUserSpaceInstrumentationEvent(
     main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kSevereWarning,
                                      GetCaptureTimeAt(error_event.timestamp_ns()), message);
     if (!IsLoadingCapture()) {
```

Comment:
Ah, another thing. I would add a comment saying that we always want to show the warning (like in the commit message). It might be confusing to read this code and notice that it's different than the one around it, which uses the "message box with check box" functionality.

Reply:
done